scripts/python_scripts/generate_network_status.py

```diff
@@ -130,24 +129,38 @@ def generate_network_list(url):
     return feature_list
 
 def check_features(chains):
-    '''
-    features may be:
-    [
-        '📚 Transfer History',
-        '✨ ORML/Assets',
-        '📈 Staking analytics',
-        '🥞 Staking rewards'
-    ]
-    '''
-    features = ['📚 Transfer History']
-    for chain in chains:
-        if (len(chain.get('assets')) > 1):
-            features.append('✨ ORML/Assets')
-        if (chain.get('assets')[0].get('staking')):
-            if (chain.get('assets')[0].get('staking') == 'relaychain'):
-                features.append('📈 Staking analytics')
-            features.append('🥞 Staking rewards')
-    return '<br />'.join(features)
+	def has_transfer_history(chain):
+		return True
+
+	def has_orml_or_asset(chain):
+		if (len(chain.get('assets')) > 1):
+			return True
+		return False
```

Comment:
I would suggest searching for `assets` or `orml` in a`ssets.type` to avoid implicit assumption that `multiple assets -> assets or orml` which might not be true in the future (for example with Darwinia Crab)